stellarphot/photometry/tests/test_profiles.py

```diff
@@ -117,27 +158,82 @@ def test_radial_profile_exposure_is_nan():
 def test_radial_profile_with_background():
     # Regression test for #328 -- image with a background level
     image = make_gaussian_sources_image(SHAPE, STARS)
+    noise_stdev = 10
     image = image + make_noise_image(
-        image.shape, distribution="gaussian", mean=100, stddev=0.1
+        image.shape, distribution="gaussian", mean=100, stddev=noise_stdev, seed=43917
     )
     for row in STARS:
         cen = find_center(image, (row["x_mean"], row["y_mean"]), max_iters=10)
 
         # The "stars" have FWHM around 9.5, so make the cutouts used for finding the
         # stars fairly big -- the bare minimum would be a radius of 3 FWHM, which is a
         # cutout size around 60.
-        rad_prof = CenterAndProfile(image, cen, cutout_size=60, profile_radius=30)
+        rad_prof = CenterAndProfile(
+            image, cen, centering_cutout_size=60, profile_radius=30
+        )
 
         # Numerical value below is integral of input 2D gaussian, 2pi A sigma^2
         expected_integral = 2 * np.pi * row["amplitude"] * row["x_stddev"] ** 2
 
+        # The standard deviation in the sum of N gaussian random variables with
+        # standard deviation SD is
+        #    σ = sqrt(N × SD^2)
+        # The curve of growth includes the sume of a bunch of pixels which each have a
+        # standard deviation of 10, so the standard deviation of the sum of those pixels
+
+        expected_stddev = np.sqrt(rad_prof.curve_of_growth.area[-1] * noise_stdev**2)
+
+        # Allow for a 3-sigma tolerance
```

Comment:
In principle this test would fail 0.3% of the time (if it is 3*sigma tolerance)... BUT if you fix the seed above, this should always work). (sorry, just trying to follow what is happening).